src/ui/server/controllers/quote/tell-us-about-your-policy/index.ts

```diff
@@ -111,15 +111,15 @@ const get = async (req: Request, res: Response) => {
     }
 
     let mappedCurrencies;
-    if (submittedData && submittedData.quoteEligibility && submittedData.quoteEligibility[CURRENCY]) {
+    if (submittedData?.quoteEligibility && submittedData.quoteEligibility[CURRENCY]) {
```

Comment:
`objectHasProperty`? :P

Reply:
Good shout, would like to do this as part of https://ukef-dtfs.atlassian.net/browse/EMS-977 if that's ok? There's a fair amount of instances where we could simplify and use `objectHasProperty`